docs/getting-started-guides/scratch.md

```diff
@@ -0,0 +1,726 @@
+pre-defined guides.
+
+This guide is also useful for those wanting to understand at a high level some of the
+steps that existing cluster setup scripts are making.
+
+**Table of Contents**
+
+- [Designing and Preparing](#designing-and-preparing)
+    - [Learning](#learning)
+    - [Cloud Provider](#cloud-provider)
+    - [Nodes](#nodes)
+    - [Network](#network)
+    - [Cluster Naming](#cluster-naming)
+    - [Software Binaries](#software-binaries)
+        - [Downloading and Extracting Kubernetes Binaries](#downloading-and-extracting-kubernetes-binaries)
+        - [Selecting Images](#selecting-images)
+    - [Security Models](#security-models)
+        - [Preparing Certs](#preparing-certs)
+        - [Preparing Credentials](#preparing-credentials)
+- [Configuring and Installing Base Software on Nodes](#configuring-and-installing-base-software-on-nodes)
+    - [Docker](#docker)
+    - [rkt](#rkt)
+    - [kubelet](#kubelet)
+    - [kube-proxy](#kube-proxy)
+    - [Networking](#networking)
+    - [Other](#other)
+    - [Using Configuration Management](#using-configuration-management)
+- [Bootstrapping the Cluster](#bootstrapping-the-cluster)
+    - [etcd](#etcd)
+    - [Apiserver](#apiserver)
+        - [Apiserver pod template](#apiserver-pod-template)
+    - [Starting Apiserver](#starting-apiserver)
+    - [Scheduler](#scheduler)
+    - [Controller Manager](#controller-manager)
+    - [DNS](#dns)
+    - [Logging](#logging)
+    - [Monitoring](#monitoring)
+    - [Miscellaneous Resources](#miscelaneous-resources)
+- [Troubleshooting](#troubleshooting)
+    - [Running validate-cluster](#running-validate-cluster)
+    - [Inspect pods and services](#inspect-pods-and-services)
+    - [Try Examples](#try-examples)
+    - [Running the Conformance Test](#running-the-conformance-test)
+    - [Networking](#networking)
+    - [Getting Help](#getting-help)
+
+## Designing and Preparing
+
+### Learning 
+  1. You should be familiar with using Kubernetes already.  We suggest you set
+    up a temporary cluster by following one of the other Getting Started Guides.
+    This will help you become familiar with the CLI ([kubectl](../kubectl.md)) and concepts ([pods](../pods.md), [services](../services.md), etc.) first.
+  1. You should have `kubectl` installed on your desktop.  This will happen as a side
+    effect of completing one of the other Getting Started Guides.
+
+### Cloud Provider
+Kubernetes has the concept of a Cloud Provider, which is a module which provides
+an interface for managing TCP Load Balancers, Nodes (Instances) and Networking Routes.
+The interface is defined in `pkg/cloudprovider/cloud.go`.  It is possible to
+create a custom cluster without implementing a cloud provider (for example if using
+bare-metal), and not all parts of the interface need to be implemented, depending
+on how flags are set on various components. 
+
+### Nodes
+- You can use virtual or physical machines.
+- The machines need to be on the same subnet.
```

Comment:
Define subnet?  Are you meaning an IP-space?  Technically we're all on a /0 subnet, I guess.  I think this statement is not clear asis.